src/App.jsx

```diff
@@ -11,6 +12,9 @@ const CHAIN_ID = 'finschia-2';
 // https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-2.md
 const CAIP_BLOCKCHAIN_ID = `cosmos:${CHAIN_ID}`
 
+const LCD_ENDPOINT= 'https://lbs-finschianw.line-apps.com'
```

Comment:
I think we need to use lbm-proxy([https://dsvt-](http://dsvt.finschia.line-apps.com/)[finschia](http://dsvt.finschia.line-apps.com/)[.line-apps.com](http://dsvt.finschia.line-apps.com/)), don't we?

Reply:
It doesn't matter which node it is because it only queries without sending tx. Which do you prefer, lbs node or dosi vault node?

Reply:
it's about DOSI Vault, so IMO, dosi vault node is preferred. :)